### DISCUSSION TITLE
Support a `code-link` option in markdown formats

### DISCUSSION THREAD

**Discussion by wurli** [quarto-dev/quarto-cli#5595] [Feature Requests]
Fairly self explanatory 😄 This option is already super handy in `html` documents, but I'd really appreciate it in markdown (and other formats!), particularly in `gfm` for writing project READMEs.

**Comment by mcanouil** [comment]
I don't know if that is something feasible or easily at least.
`code-link` is done by `downlit` R package thus only for R code via the `knitr` engine.
`code-link` has already several limitations with other features that actually work not only for R.

**Reply by mcanouil** [reply]
FYI:

- #3777

Closing this thread as a duplicate.
